Base/QTCore/qSlicerExtensionsManagerModel.cxx

```diff
@@ -481,19 +482,23 @@ QStringList removeFromPathList(const QStringList& paths, const QStringList& path
 void qSlicerExtensionsManagerModelPrivate::addExtensionPathToApplicationSettings(const QString& extensionName)
 {
   Q_Q(qSlicerExtensionsManagerModel);
-  QSettings settings(q->extensionsSettingsFilePath(), QSettings::IniFormat);
-  QStringList additionalPaths = settings.value("Modules/AdditionalPaths").toStringList();
-  settings.setValue("Modules/AdditionalPaths",
+  ctkCoreSettings settings(q->extensionsSettingsFilePath(), QSettings::IniFormat);
+  settings.setApplicationHomeDirectory(qSlicerCoreApplication::application()->slicerHome());
+  settings.setApplicationHomePlaceholder("<APPLAUNCHER_DIR>");
+  QStringList additionalPaths = settings.pathValue("Modules/AdditionalPaths").toStringList();
+  settings.setPathValue("Modules/AdditionalPaths",
                     appendToPathList(additionalPaths, q->extensionModulePaths(extensionName)));
 }
 
 // --------------------------------------------------------------------------
 void qSlicerExtensionsManagerModelPrivate::removeExtensionPathFromApplicationSettings(const QString& extensionName)
 {
   Q_Q(qSlicerExtensionsManagerModel);
-  QSettings settings(q->extensionsSettingsFilePath(), QSettings::IniFormat);
-  QStringList additionalPaths = settings.value("Modules/AdditionalPaths").toStringList();
-  settings.setValue("Modules/AdditionalPaths",
+  ctkCoreSettings settings(q->extensionsSettingsFilePath(), QSettings::IniFormat);
+  settings.setApplicationHomeDirectory(qSlicerCoreApplication::application()->slicerHome());
+  settings.setApplicationHomePlaceholder("<APPLAUNCHER_DIR>");
```

Comment:
Consider using `<APPLAUNCHER_SETTINGS_DIR>` , also the `ApplicationHomeDirectory` may have to be set to match the location of the settings file instead.

Reply:
To ensure `PythonSlicer` works, could you updated to use `APPLAUNCHER_SETTINGS_DIR` ?